<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tfs::Internal::TfsWorkspaceDialog</class>
 <widget class="QDialog" name="Tfs::Internal::TfsWorkspaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Showing workspace on this computer to which you have access.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Workspaces:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>20000</width>
       <height>20000</height>
      </size>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Show remotes workspaces</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdAdd">
       <property name="text">
        <string>Add...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdEdit">
       <property name="text">
        <string>Edit...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdClose</sender>
   <signal>clicked()</signal>
   <receiver>Tfs::Internal::TfsWorkspaceDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
